scanapp/models.py

```diff
@@ -102,6 +102,87 @@ def __str__(self):
 
     scan = models.ForeignKey(Scan)
     path = models.CharField(max_length=400, help_text='Path of file scanned')
+    type = models.CharField(max_length=400, help_text='Type of the entity being scanned')
+    name = models.CharField(max_length=400, help_text='Name of the entity being scanned')
+    base_name = models.CharField(
+        max_length=400,
+        help_text='Base name of entity without extension',
+    )
+    extension = models.CharField(
+        max_length=400,
```

Comment:
400 seems to be big for a field that just stores the file extension. This can be changed to something smaller. You should also take a look at other field sizes and make them smaller.

Reply:
OK.